tests/test_expectation_maximization.py

```diff
@@ -0,0 +1,173 @@
+"""Test bayesian expectation maximization process."""
+
+import numpy as np
+import pytest
+
+from reconstructSPI.iterative_refinement import expectation_maximization as em
+
+
+@pytest.fixture
+def n():
```

Comment:
`n` --> `n_pix`

Reply:
This was left as n to encompass both n_particles and n_pix for a super simplified testing environment. You're completely right in that this isn't a great design choice, so I "split" the value into n_particles and n_pix and updated the tests accordingly.